reana_db/version.py

```diff
@@ -14,4 +14,4 @@
 
 from __future__ import absolute_import, print_function
 
-__version__ = "0.7.0.dev20200619"
+__version__ = "0.7.0.dev20200708"
```

Comment:
Can you please remove the commit `release: 0.7.0.dev20200708` commit and rather add a new one on the branch head that would be `0.7.0a1` version already?

Question: (a) Are we ready to start tagging these alpha releases properly? Which would mean to update release notes, to mint github tag, etc? (b) Or are we OK to still not tag these properly?  Which means just to release alpha1 on PyPY but not separating release notes and not minting git tags? I think the latter option (b) might do for speed purposes, since alphas are still meant to be pretty much internal.

Reply:
I would go for (b) as it is more gradual.